listener/subgraph_definition/schema.graphql

```diff
@@ -23,6 +23,14 @@ type Withdraw @entity {
     slotIndex: Int!
 }
 
+type StandingSellOrderBatch @entity {
+    id: ID!
+    accountId: Int!
+    batchIndex: BigInt!
+    validFromAuctionId: BigInt!
+    orders: [SellOrder!]
```

Comment:
auctionId and slotIndex on the SellOrder entity might have to become optional (as in we need to remove the !)